### PR TITLE
tentative solution

### PR DESCRIPTION
**Pull request by Dasha529**
it is necessary to complete the logic 
issue #86

### CODE REVIEW

**Comment by AVLozhechkin** [comment]
изображение
There is a problem with input lines after incorrect input.

File: HW3/Fedushina Daria/ConsoleDrawGame/Board.cs [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    Vertical = 20;
+                else
+                    Vertical = value;
+            }
+        }
+
+
+        public void Draw(IBoard board)
+        {
+            OriginalX = OrigX;
```
Comment (bartimys1) [inline HW3/Fedushina Daria/ConsoleDrawGame/Board.cs]: u have property 

> public int OrigX 

in set use 

> OriginalX = value;

in get use

> return OriginalX;

and after

> OrigX = ioDevice.GetCursorPosition().Item1;

File: HW3/Fedushina Daria/ConsoleDrawGame/Game.cs [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+        private delegate void Draw(IBoard board);
+
+        public void Run()
+        {
+            string userNumber="";
+            int FigureCount = 1;
+            int[] NumArray = { 1, 2, 3, 4 };
+            List<int> userArray = new List<int> { 1 };
+            IBoard ConsoleBoard = new Board();
```
Comment (bartimys1) [inline HW3/Fedushina Daria/ConsoleDrawGame/Game.cs]: you already have

>  private readonly IBoard board;

File: HW3/Fedushina Daria/ConsoleDrawGame/IBoard.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+        int boardSizeX { get; set; }
+        int boardSizeY { get; set; }
+
+        int OrigX { get; set; }
+        int OrigY { get; set; }
+
+        void WriteAt(string symbol, int x, int y);
+
+        void Draw(IBoard board);
```
Comment (bartimys1) [inline HW3/Fedushina Daria/ConsoleDrawGame/IBoard.cs]: it is very strong) your board can draw on the “other” board

File: HW3/Fedushina Daria/ConsoleDrawGame/Resources/gamesettings.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿{
+  "BackDoor": 0,
+  "DoorsAmount": 5,
```
Comment (MilRoad) [inline HW3/Fedushina Daria/ConsoleDrawGame/Resources/gamesettings.json]: Is it from last game? :D

File: HW3/Fedushina Daria/ConsoleDrawGame/FigureDrawing.cs [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+namespace ConsoleDrawGame
+{
+    class FigureDrawing : IFigureDrawing
+    {
+        int OriginalX;
+        int OriginalY;
+        int BenderCount;
+        ConsoleIO ioDevice = new ConsoleIO();
+        //(int, int) OrigXY;
+        int IFigureDrawing.Bender
```
Comment (MilRoad) [inline HW3/Fedushina Daria/ConsoleDrawGame/FigureDrawing.cs]: I didn't get what is it. And I don't understand the translation of word Bender

File: HW3/Fedushina Daria/ConsoleDrawGame/IBoard.cs (hunk 3 above)
Comment (MilRoad) [reply HW3/Fedushina Daria/ConsoleDrawGame/IBoard.cs]: I think that's for using it with delegate

**Comment by KorolS** [comment]
I have added a comment https://github.com/chesdenis/net-courses-2019/pull/146#issuecomment-516594090 to new pullrequest

**Pull request closed without merging**